### PR TITLE
Remove license headers left over

### PR DESCRIPTION
**Pull request by guillaumerose**

### CODE REVIEW

File: cmd/crc/cmd/console.go [hunk 1]
```diff
@@ -1,16 +1,3 @@
-/*
-Copyright 2016 The Kubernetes Authors All rights reserved.
```
Comment (gbraad) [inline cmd/crc/cmd/console.go:2]: Not sure why this mentions kubernetes?!? C&P issue?

**Comment by guillaumerose** [comment]
I added the missing headers and a CI step to verify we don't miss one.

**Comment by praveenkumar** [comment]
@gbraad @guillaumerose can't we remove the license headers from all the files since we already have a license file or these headers means something different?

**Pull request merged**